mass_editing/static/description/index.html

```diff
@@ -0,0 +1,68 @@
+<section class="oe_container oe_dark">
```

Comment:
This should be included in the README.rst instead of in this HTML field. This way, everyone can see the guide.